.github/workflows/rpcs3.yml

```diff
@@ -312,40 +312,48 @@ jobs:
         run: .ci/github-upload.sh
 
       - name: Save Build Ccache
-        if: github.ref == 'refs/heads/master'
+        #if: github.ref == 'refs/heads/master'
         uses: actions/cache/save@main
         with:
           path: ${{ env.CCACHE_DIR }}
           key: ${{ steps.restore-build-ccache.outputs.cache-primary-key }}
 
       - name: Save Dependencies Cache
-        if: github.ref == 'refs/heads/master'
+        #if: github.ref == 'refs/heads/master'
         uses: actions/cache/save@main
         with:
           path: ${{ env.DEPS_CACHE_DIR }}
           key: ${{ steps.restore-dependencies-cache.outputs.cache-primary-key }}
 
   Windows_Build_Clang:
     # Only run push event on master branch of main repo, but run all PRs
-    if: github.event_name != 'push' || (github.repository == 'RPCS3/rpcs3' && github.ref_name == 'master')
-    name: RPCS3 Windows Clang
+    #if: github.event_name != 'push' || (github.repository == 'RPCS3/rpcs3' && github.ref_name == 'master')
     runs-on: windows-2025
     strategy:
+      fail-fast: false
       matrix:
         include:
           - msys2: clang64
             compiler: clang
             arch: win64
+          - compiler: clang-cl
+            llvmver: 20.1.8
```

Comment:
Why is this not using 19.1.7 as the other builds?

Reply:
The windows image runner uses 20.1.8. So this is matching that. It can be set to use 19.1.7 if preferable.